src/multipart_stream.cpp

```diff
@@ -16,7 +16,8 @@ void MultipartStream::sendInitialHeader() {
       "Server", "web_video_server").header("Cache-Control",
                                            "no-cache, no-store, must-revalidate, pre-check=0, post-check=0, max-age=0").header(
       "Pragma", "no-cache").header("Content-type", "multipart/x-mixed-replace;boundary="+boundry_).header(
-      "Access-Control-Allow-Origin", "*").write(connection_);
+      "Access-Control-Allow-Origin", "*").header("Access-Control-Allow-Methods", "GET,POST,PUT,DELETE,HEAD,OPTIONS").header(
```

Comment:
I have a hunch that POST, PUT, and DELETE do not need to be in this list.  Does it work without them?

Reply:
Sorry for the late reply.

I checked whether it works according to value of "Access-Control-Allow-Methods".

POST and DELETE do not need to be in this list.
On the other hand, PUT is needed to be in this list.
I checked the request header from DepthCloud's video tag in Ros3d.js.
"GET" was specified in HTTP method for request video stream.
I can't understand logically because PUT method was not used in HTTP request.

It seems reasonable to specify "GET,POST,PUT,HEAD,OPTIONS" (excluding DELETE)
for Access-Control-Allow-Methods.
How about this proposal?

Reply:
Interesting, sounds good.